js/adapt-mediaWithBenefits.js

```diff
@@ -1,184 +1,185 @@
+
+      // update classes
+      this.$('.js-mediawithbenefits-item').removeClass('is-selected');
+      $selectedItem.addClass('is-selected');
+      $selectedItem.addClass('is-visited');
+
+      this.selectedIndex = index;
+
+      // TODO: add support for Youtube/Vimeo sources
+      this.mediaElement.setSrc(itemCfg._media.mp4);
+      this.mediaElement.load();
+
+      // set up transcript
+      if (itemCfg._transcript) {
+        this.model.set('_transcript', _.extend({}, this.model.get('_originalTranscript'), itemCfg._transcript));
+        this.$('.media__transcript-container').html(Handlebars.partials.transcript(this.model.toJSON()));
+      } else {
+        this.model.unset('_transcript');
+        this.$('.media__transcript-container').empty();
+      }
+
+      // setup poster image
+
+      $(this.mediaElement).attr('poster', itemCfg._media.poster);
+      this.mediaElement.player.setPoster(itemCfg._media.poster);
+
+      var selectedTrack = this.mediaElement.player.selectedTrack;
+      var selectedLang = selectedTrack ? selectedTrack.srclang || 'none' : 'none';
+
+      // restore last language selection if applicable
+      this.mediaElement.player.options.startLanguage = selectedLang;
+
+      $(this.mediaElement).find('track').remove();
+      $(this.mediaElement).append(Handlebars.partials.mediaTracks(itemCfg._media.cc));
+      this.mediaElement.player.rebuildtracks();
+    },
+
+    playSelection:function() {
+      var $selectedItem = this.$('.js-mediawithbenefits-item').eq(this.selectedIndex);
+
+      this.mediaElement.play();
+
+      $selectedItem.addClass('is-visited');
+    },
+
+    onItemClicked:function(e) {
+      var index = $(e.currentTarget).data('index');
+
+      if (this.selectedIndex != index) {
+        this.selectItem(index);
+      }
+
+      this.playSelection();
+    },
+
+    onToggleInlineTranscript: function(event) {
+      this.markItemAsWatched();
+
+      if (event) event.preventDefault();
+      var $transcriptBodyContainer = this.$(".media__transcript-body-inline");
+      var $button = this.$(".media__transcript-btn-inline");
+      var $buttonText = this.$(".media__transcript-btn-inline .media__transcript-btn-text");
+
+      if ($transcriptBodyContainer.hasClass("inline-transcript-open")) {
+        $transcriptBodyContainer.stop(true,true).slideUp(function() {
+          $(window).resize();
+        });
+        $button.attr('aria-expanded', false);
+        $transcriptBodyContainer.removeClass("inline-transcript-open");
+        $buttonText.html(this.model.get("_transcript").inlineTranscriptButton);
+      } else {
+        $transcriptBodyContainer.stop(true,true).slideDown(function() {
+          $(window).resize();
+        });
+        $button.attr('aria-expanded', true);
+        $transcriptBodyContainer.addClass("inline-transcript-open");
+        $buttonText.html(this.model.get("_transcript").inlineTranscriptCloseButton);
+
+        this.checkCompletionByTranscript();
+      }
```

Comment:
```suggestion
      var $transcriptBodyContainer = this.$('.media__transcript-body-inline');
      var $button = this.$('.media__transcript-btn-inline');
      var $buttonText = $button.find('.media__transcript-btn-text');

      if ($transcriptBodyContainer.hasClass('inline-transcript-open')) {
        $transcriptBodyContainer.stop(true,true).slideUp(function() {
          $(window).resize();
        }).removeClass('inline-transcript-open');
        $button.attr('aria-expanded', false);
        $buttonText.html(this.model.get('_transcript').inlineTranscriptButton);
        return;
      }
      
      $transcriptBodyContainer.stop(true,true).slideDown(function() {
        $(window).resize();
      }).addClass('inline-transcript-open');
      $button.attr('aria-expanded', true);
      $buttonText.html(this.model.get('_transcript').inlineTranscriptCloseButton);

      this.checkCompletionByTranscript();
```

Reply:
I have edited the js as per the suggestion above but left the if else statement in it's current form as changing it caused the closed transcript button text not to appear.